plugins/modules/packaging/language/maven_artifact.py

```diff
@@ -150,6 +150,16 @@
         default: 'md5'
         choices: ['md5', 'sha1']
         version_added: 3.2.0
+    unredirected_headers:
+        type: list
+        default: ['Authorization', 'Cookie']
+        elements: str
+        version_added: 5.2.0
+        description:
+            A list of headers that should not be included in the redirection. This headers are sent to the fetch_url
+            ``fetch_url`` function.
+            Useful if the redirection URL does not need to have sensitive headers in the request.
+            Requires Ansible version >=2.12
```

Comment:
There is no Ansible version 2.12. You mean ansible-core.
```suggestion
            Requires ansible-core version 2.12 or later.
```